2025-05-17 17:21:05,772 INFO: Добавлена заявка: {'type': 'Заселение', 'status': 'Создана', 'student_id': '21', 'student_fio': 'Серега', 'room_id': '101', 'id': '52', 'created_at': '17.05.2025 17:21'}
2025-05-17 17:21:11,634 INFO: Соединение с базой данных закрыто
2025-05-17 20:11:39,466 INFO: Успешное подключение к базе данных
2025-05-17 20:12:12,319 INFO: Применён фильтр: М
2025-05-17 20:12:12,542 INFO: Применён фильтр: Ма
2025-05-17 20:12:12,684 INFO: Применён фильтр: Мат
2025-05-17 20:12:13,620 INFO: Применён фильтр: Ма
2025-05-17 20:12:13,761 INFO: Применён фильтр: М
2025-05-17 20:12:14,027 INFO: Применён фильтр: 
2025-05-17 20:12:14,514 INFO: Применён фильтр: Ф
2025-05-17 20:12:14,819 INFO: Применён фильтр: ФМ
2025-05-17 20:12:14,980 INFO: Применён фильтр: ФМИ
2025-05-17 20:12:16,094 INFO: Применён фильтр: ФМ
2025-05-17 20:12:16,916 INFO: Применён фильтр: Ф
2025-05-17 20:12:16,916 INFO: Применён фильтр: 
2025-05-17 20:12:17,871 INFO: Применён фильтр: 1
2025-05-17 20:12:18,061 INFO: Применён фильтр: 10
2025-05-17 20:12:18,156 INFO: Применён фильтр: 101
2025-05-17 20:12:19,047 INFO: Применён фильтр: 10
2025-05-17 20:12:19,206 INFO: Применён фильтр: 1
2025-05-17 20:12:20,243 INFO: Применён фильтр: 11
2025-05-17 20:12:20,386 INFO: Применён фильтр: 11-
2025-05-17 20:12:20,833 INFO: Применён фильтр: 11
2025-05-17 20:12:21,204 INFO: Применён фильтр: 1
2025-05-17 20:12:21,408 INFO: Применён фильтр: 10
2025-05-17 20:12:21,531 INFO: Применён фильтр: 102
2025-05-17 20:12:22,192 INFO: Применён фильтр: 10
2025-05-17 20:12:22,352 INFO: Применён фильтр: 1
2025-05-17 20:12:22,495 INFO: Применён фильтр: 
2025-05-17 20:12:24,710 INFO: Соединение с базой данных закрыто
2025-05-17 20:16:17,697 INFO: Успешное подключение к базе данных
2025-05-17 20:17:16,653 INFO: Добавлен студент ID 31: {'fio': 'Анатлий П', 'pol': 'М', 'vozrast': 22, 'kurs': 4, 'fakultet': 'ХЗ', 'number_phone': '', 'id': '31', 'room_id': 'еще не заселен'}
2025-05-17 20:23:27,565 INFO: Соединение с базой данных закрыто
2025-05-17 21:08:16,822 INFO: Успешное подключение к базе данных
2025-05-17 21:08:31,406 INFO: Соединение с базой данных закрыто
2025-05-17 21:40:47,058 INFO: Успешное подключение к базе данных
2025-05-17 21:47:27,221 INFO: Соединение с базой данных закрыто
2025-05-19 13:53:45,351 ERROR: Ошибка подключения к базе данных: 'utf-8' codec can't decode byte 0xdd in position 81: invalid continuation byte
2025-05-19 13:53:45,724 ERROR: Неперехваченное исключение
Traceback (most recent call last):
  File "dip.py", line 125, in <module>
  File "dip.py", line 74, in __init__
  File "dip.py", line 78, in connect
  File "psycopg2\__init__.py", line 122, in connect
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdd in position 81: invalid continuation byte
2025-05-19 13:54:02,099 ERROR: Ошибка подключения к базе данных: 'utf-8' codec can't decode byte 0xdd in position 81: invalid continuation byte
2025-05-19 13:54:02,099 ERROR: Неперехваченное исключение
Traceback (most recent call last):
  File "dip.py", line 125, in <module>
  File "dip.py", line 74, in __init__
  File "dip.py", line 78, in connect
  File "psycopg2\__init__.py", line 122, in connect
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdd in position 81: invalid continuation byte
2025-05-19 13:54:58,260 ERROR: Ошибка подключения к базе данных: connection to server at "aws-0-eu-north-1.pooler.supabase.com" (13.48.169.15), port 6543 failed: timeout expired
connection to server at "aws-0-eu-north-1.pooler.supabase.com" (16.16.102.12), port 6543 failed: timeout expired

2025-05-19 13:54:58,260 ERROR: Неперехваченное исключение
Traceback (most recent call last):
  File "dip.py", line 125, in <module>
  File "dip.py", line 74, in __init__
  File "dip.py", line 78, in connect
  File "psycopg2\__init__.py", line 122, in connect
psycopg2.OperationalError: connection to server at "aws-0-eu-north-1.pooler.supabase.com" (13.48.169.15), port 6543 failed: timeout expired
connection to server at "aws-0-eu-north-1.pooler.supabase.com" (16.16.102.12), port 6543 failed: timeout expired

2025-05-19 13:55:30,570 ERROR: Ошибка подключения к базе данных: 'utf-8' codec can't decode byte 0xdd in position 81: invalid continuation byte
2025-05-19 13:55:30,570 ERROR: Неперехваченное исключение
Traceback (most recent call last):
  File "dip.py", line 125, in <module>
  File "dip.py", line 74, in __init__
  File "dip.py", line 78, in connect
  File "psycopg2\__init__.py", line 122, in connect
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdd in position 81: invalid continuation byte
2025-05-19 13:55:49,952 INFO: Успешное подключение к базе данных
2025-05-19 13:56:30,262 INFO: Применён фильтр: А
2025-05-19 13:56:31,296 INFO: Применён фильтр: Ар
2025-05-19 13:56:31,482 INFO: Применён фильтр: Арт
2025-05-19 13:56:33,301 INFO: Применён фильтр: Ар
2025-05-19 13:56:35,156 INFO: Применён фильтр: А
2025-05-19 13:56:35,159 INFO: Применён фильтр: 
2025-05-19 13:59:16,058 INFO: Соединение с базой данных закрыто
